pkg/controller/computenodeopenstack/computenodeopenstack_controller.go

```diff
@@ -358,6 +358,10 @@ func getRenderData(ctx context.Context, client client.Client, instance *computen
 	data.Data["K8sServiceIP"] = instance.Spec.K8sServiceIP
 	data.Data["APIIntIP"] = instance.Spec.APIIntIP
 	data.Data["Workers"] = instance.Spec.Workers
+	data.Data["Dedicated"] = true
+	if instance.Spec.Dedicated != "" {
```

Comment:
instance.Spec.Dedicated is bool, so this should be `instance.Spec.Dedicated != false`, no? Thats also why the check fails

Reply:
right!